Documentation/running-clair.md

```diff
@@ -44,14 +44,15 @@ A [PostgreSQL 9.4+] database instance is required for all instructions.
 #### Kubernetes
 
 If you don't have a local Kubernetes cluster already, check out [minikube].
+This also requires [helm].
```

Comment:
How does something like this sound instead?
>This assumes you've already ran `helm init` and you have access to a currently running instance of Tiller.

Reply:
good point